src/MusicStore/Components/SystemClock.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace MusicStore.Components
+{
+    /// <summary>
+    /// Provides access to the normal system clock.
+    /// </summary>
+    public class SystemClock : ISystemClock
+    {
+        /// <inheritdoc />
+        public DateTime UtcNow
+        {
+            get
+            {
+                // The clock measures whole seconds only, to have integral expires_in results, and
```

Comment:
Is this milliseconds stuff needed here? I think the Authentication version of this had that because the time gets persisted into various other places such as cookies or something.